src/kzg10/mod.rs

```diff
@@ -18,6 +20,8 @@ use rayon::prelude::*;
 
 mod data_structures;
 pub use data_structures::*;
+mod subproductdomain;
+pub use subproductdomain::*;
```

Comment:
Stylistic nit, but usually Rust uses snake-case for module names, which means this should be something like

```suggestion
mod subproduct_domain;
pub use subproduct_domain::*;
```